versions.tf

```diff
@@ -0,0 +1,13 @@
+terraform {
+  required_version = ">= 1.3.0"
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = ">= 4.0"
+    }
+    random = {
```

Comment:
@DCamma Is there a specific requirement to keep this provider version fixed to `3.4.3`. It could conflict with other libraries if they fix it to something else (e.g. `3.4.2`)